### ISSUE TITLE
Notify about Calls from VoIP Apps

### ISSUE THREAD

**Issue by veecue** [Freeyourgadget/Gadgetbridge#1393]
#### Your issue is:
I would like to get call notifications about VoIP calls via IMs (e.g. Telegram, Threema, WhatsApp) and via specialized Apps (like FRITZ! Box Fon). A bit of research shows, that most of those apps Send an ongoing notification with an Accept and Reject PendingIntent that could be parsed. Telegram however only does so, when the device is unlocked and displays a custom screen when the device is locked. Any ideas on how to solve this?

#### Your wearable device is:
Mi Band 3

*Please specify model and firmware version if possible*

#### Your android version is:
7.0

#### Your Gadgetbridge version is:
master

*New requests about already solved/documented topics could be closed without further comments. Same for too generic or incomplete reports.*

**Comment by ashimokawa** [comment]
Yep. Agree. Problem is that when scraping button captions we have to support tons of languages. 😒

Or is there a way to get untranslated strings? I guess no ...

**Comment by veecue** [comment]
I think that parsing the button order would be the most reliable thing to do (per app :(). Do you have an idea on how to get something working with Telegram? I will start writing an implementation that catches ongoing Notifications of type "call" and tries to extract the reject button's Intent (the mi band 3 can't accept, so for now, I'll start with reject)

**Comment by MMurzik** [comment]
I'd like call notification from Zoiper pro VoIP app too

**Comment by veecue** [comment]
@MMurzik can you check out my proof of concept code (declining doesn't work yet) and test it with your app? Would be cool if you could submit the debug output aswell
